content/rc/how-to/aws-zone-mapping.md

```diff
@@ -1,84 +1,68 @@
 
-The Availability Zones may be named the same for different AWS accounts,
-but under the hood are actually resides on different infrastructure.
-This can cause latency overhead when your app and your database are not
-located on the same physical infrastructure. Since your app runs under
-your AWS account and your Redis Cloud Essentials databases is under Redis Labs' AWS account,
-you may suffer from this issue.
+The Availability Zones names can be the same for different AWS accounts,
+but are actually hosted in different locations.
+This can cause latency overhead when your app and your database are not located on the same physical infrastructure. Because your application runs in your AWS account and your Redis Cloud Essentials databases runs in the Redis Labs AWS account,
+the AZs can be hosted in different locations and cause latency overhead.
 
-To overcome this issue, we came up with an automatic process that can
-map AZs that are designated to your AWS account and cross-reference them
-with our AZ. This allows us to make sure your app and database are co-located on the same physical AZ
-and by that reducing unnecessary latency.
+To mitigate for this latency overhead, you can automatically map AZs that are designated to your AWS account
+with the AZs in your Redis Cloud account.
+This makes sure that your application and database are co-located on the same physical AZ.
 
 ## How to MAP Zones
 
-1. [Log in](https://app.redislabs.com/#/login) to your Redis Cloud Essentials account
-1. Navigate to **Settings** page from the menu and then the **Account**
-    tab
-1. Press the **Map your AWS Zones** button
-1. Enter the Access Key ID and Secret Access Key for an [unprivileged
-    AWS user](#unprivileged-user-creation).
-1. Press on **Map Zones** button.
+1. [Log in](https://app.redislabs.com/#/login) to your Redis Cloud Essentials account.
+1. Go to **settings** > **account**.
+1. Click **Map your AWS Zones**.
+1. Enter the Access Key ID and Secret Access Key for an [unprivileged AWS user](#unprivileged-user-creation).
+1. Click **Map Zones**.
 
-Allow the process a few minutes to complete as it maps the availability
-zones. Once complete, you see this:
+The zone mapping takes a few minutes to complete as it maps the availability zones.
+During the zone mapping you see:
 
 ![zone-mapping](/images/rc/zone-mapping.png?width=600&height=359)
 
-When the process is complete, the next time you go to create a new
-subscription, the drop down for "Cloud" has the zone letter in
-it.
+After zone mapping is complete, the dropdown for **Cloud** in a new subscription shows the zone letter.
 
 ![after_zone_mapping](/images/rc/after_zone_mapping.png?width=600&height=192)
 
-For existing subscriptions in Redis Cloud Essentials, you now see the Availability Zone
-they are using. To reduce the latency described above:
+For existing subscriptions in Redis Cloud Essentials, you now see the Availability Zone they are using.
+To reduce the latency described above:
```

Comment:
I thought reducing latency was the goal of the previous procedure. What additional benefit is gained from creating a new subscription, database, and migration of data to the new db? Sounds like this would take considerably longer.